extensions/wikia/WallNotifications/GlobalNavigationWallNotificationsController.class.php

```diff
@@ -6,11 +6,23 @@
 
 
 class GlobalNavigationWallNotificationsController extends WallNotificationControllerBase {
-	protected function addAssets() {
-		$this->response->addAsset( 'extensions/wikia/GlobalNavigation/styles/GlobalNavigationNotifications.scss' );
-		$this->response->addAsset( 'wall_notifications_global_navigation_js' );
+	const NOTIFICATION_TITLE_LIMIT = 48;
+
+	private $wallHelper;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wallHelper" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1068)